code/modules/power/supermatter/supermatter.dm

```diff
@@ -125,8 +128,11 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/tritiumcomp = 0
 	var/bzcomp = 0
 	var/pluoxiumcomp = 0
+	var/h2ocomp = 0
 
 	var/pluoxiumbonus = 0
+	var/h2obonus = 0
+	var/h2omalus = 1
```

Comment:
could you add some comments either here or down where it's set on what this variable is used for?